sdk/ai/azure-ai-generative/assets.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
What is this file for?

Reply:
It instructs the test-proxy to store recordings in an external repository